pack/df.json

```diff
@@ -909,7 +909,7 @@
         "deck_limit": 3,
         "faction_code": "nbn",
         "faction_cost": 1,
-        "flavor": "Though many countries attempted to regulate digital loot boxes in the early 21st century, GameNET has managed to circumvent any such laws via explicit, transparent percentage rates...and some very determined lobbyists.",
+        "flavor": "Though many countries attempted to regulate digital loot boxes in the early 21st century, GameNET has managed to circumvent any such laws via explicit, transparent percentage rates... and some very determined lobbyists.",
```

Comment:
That's odd, because if there were no space after the ellipsis, I expect the line wouldn't break where it does on the card.

Also, I totally disagree with that from a stylistic perspective (ellipses are always followed by spaces!), but I suppose that's neither here nor there.

Reply:
Yeah, I had the same view, which is why I confirmed it with them, and they specified no space, so I guess we should take that as canon, even if we disagree. 😄